uui/components/inputs/Switch.module.scss

```diff
@@ -36,14 +36,14 @@
     --uui-switch-bg-checked-hover: var(--uui-color-60);
     --uui-switch-bg-checked-disabled: var(--uui-color-20);
     --uui-switch-border: var(--uui-control-border);
-    --uui-switch-border-hover: var(--uui-neutral-40);
+    --uui-switch-border-hover: var(--uui-control-bg-hover);
     --uui-switch-border-checked: var(--uui-color-50);
     --uui-switch-border-disabled: var(--uui-control-border-disabled);
     --uui-switch-border-checked-hover: var(--uui-color-60);
     --uui-switch-border-checked-disabled: var(--uui-color-20);
     --uui-switch-label: var(--uui-control-label);
     --uui-switch-label-disabled: var(--uui-control-label-disabled);
-    --uui-switch-toggler-bg: var(--uui-surface-main);
+    --uui-switch-toggler-bg: var(--uui-neutral-0);
```

Comment:
can we use --uui-control-bg here?

Reply:
We need #FFFFFF color in all themes by design, but --uui-control-bg =--uui-neutral-70 in DARK theme, so we can't.